cmd/nerdctl/main.go

```diff
@@ -152,6 +154,8 @@ func initRootCmdFlags(rootCmd *cobra.Command, tomlPath string) error {
 	rootCmd.PersistentFlags().Bool("insecure-registry", cfg.InsecureRegistry, "skips verifying HTTPS certs, and allows falling back to plain HTTP")
 	// hosts-dir is defined as StringSlice, not StringArray, to allow specifying "--hosts-dir=/etc/containerd/certs.d,/etc/docker/certs.d"
 	rootCmd.PersistentFlags().StringSlice("hosts-dir", cfg.HostsDir, "A directory that contains <HOST:PORT>/hosts.toml (containerd style) or <HOST:PORT>/{ca.cert, cert.pem, key.pem} (docker style)")
+	// Experimental is enable experimental feature, see in https://github.com/containerd/nerdctl/blob/master/docs/experimental.md
+	rootCmd.PersistentFlags().Bool("experimental", cfg.Experimental, "Enable experimental feature or not, see in https://github.com/containerd/nerdctl/blob/master/docs/experimental.md")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().Bool("experimental", cfg.Experimental, "Enable experimental features")
```

Reply:
Also consider using `AddPersistentStringFlag()` with an env var like `$NERDCTL_EXPERIMENTAL`

Reply:
because experimental is bool, add method `AddPersistentBoolFlag` method